BabbleApp/camera.py

```diff
@@ -312,31 +314,29 @@ def start_serial_connection(self, port):
         if self.serial_connection is not None and self.serial_connection.is_open:
             # Do nothing. The connection is already open on this port.
             if self.serial_connection.port == port:
-                return
+                return False
```

Comment:
I'm debating the nature of trues/falses in this method. For instance, what does a "false" convey? We're already connected and we don't need to do anything, or there was an error while opening the camera? How can we elaborate on this, maybe we could return an enum instead? Just a thought.

Reply:
Comments would help here, perhaps a block at the top of the method?

Reply:
Nvm should be good :)